swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -45,16 +61,24 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
 
   override func startTunnel(
     options: [String: NSObject]?,
-    completionHandler: @escaping (Error?) -> Void
+    completionHandler: @escaping ((any Swift.Error)?) -> Void
   ) {
+    Log.log("startTunnel called")
     super.startTunnel(options: options, completionHandler: completionHandler)
 
     // Dummy start to get the extension running on macOS after upgrade
     if options?["dryRun"] as? Bool == true {
+      Log.info("Dry run startup requested - extension awakened but not starting tunnel")
       completionHandler(nil)
       return
     }
 
+    // Log version on actual tunnel start
+    let version =
```

Comment:
I noticed the logic for getting the bundle version is duplicated - would it make sense to consolidate this to the SharedHelpers file?